src/index.ts

```diff
@@ -50,6 +49,8 @@ class ErrorWithCode extends Error {
   }
 }
 
+let getPem: (filename: string) => Promise<string>;
```

Comment:
Should the type be `(...) -> Promise<string> | undefined`?

Reply:
I don't think so, this is the type of the getPem method from the module. Will double check :)

Reply:
I should've said `((...) -> Promise<string>) | undefined`.

In other words, `getPem` is initialized only after `google-p12-pem` is loaded. Before that, it is undefined and we need to make sure the type checker detects a programming error where `getPem` is referenced before it's initialized.

Reply:
Ahhh, I get it!  Great call.  I'm surprised that having an uninitialized declaration doesn't assume this in the compiler.  Updated 👍